src/battle/area/omo2/actor/boss_common.h

```diff
@@ -0,0 +1,49 @@
+#ifndef _GENERAL_GUY_COMMON_H_
+#define _GENERAL_GUY_COMMON_H_
+
+#include "mapfs/omo_bt07_shape.h"
+
+enum N(BattleCommon) {
```

Comment:
`=` alignment is inconsistent here

Reply:
The intent is to add an additional indent level for the enumerated values of each actor var.

Reply:
ah, I see! 👍